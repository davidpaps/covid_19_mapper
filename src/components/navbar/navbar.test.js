import * as React from 'react';
import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import Header from './navbar'
import renderer from 'react-test-renderer';

describe("Navbar Component", () => {
  it('does not crash when loaded onto page', () => {
    shallow(<Header total={[]} />)
  })
  it('render correctly Header component', () => {
    const TextInputComponent = renderer.create(<Header total={[]} globalCFR={[2.9587486]} totalInt={[]} countries={[]} integerCountries={[]} />).toJSON();
    expect(TextInputComponent).toMatchSnapshot();
  });
})