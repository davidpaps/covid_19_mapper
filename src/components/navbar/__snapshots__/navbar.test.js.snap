// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component render correctly Header component 1`] = `
<nav
  className="navbar-dark bs-navbar-collapse navbar navbar-expand navbar-light fixed-top"
>
  <span
    className="navbar-brand"
  >
    <img
      alt=""
      className="d-inline-block align-top"
      height="30"
      src="https://cdn3.iconfinder.com/data/icons/science-116/64/virus-lab-scientist-biology-cell-medical-512.png?v=2"
      width="30"
    />
     
    Covid-19
  </span>
  <div
    className="dropdown"
    onKeyDown={[Function]}
  >
    <button
      aria-expanded={false}
      aria-haspopup={true}
      className="dropdown-toggle btn btn-outline-warning"
      disabled={false}
      id="last_updated"
      onClick={[Function]}
      type="button"
    >
      <span
        aria-hidden="true"
        className="spinner-grow spinner-grow-sm"
        role="status"
      />
    </button>
  </div>
  <div
    className="ml-auto navbar-nav"
    onKeyDown={[Function]}
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-outline-success"
        disabled={false}
        id="last_updated"
        onClick={[Function]}
        type="button"
      >
        <span
          aria-hidden="true"
          className="spinner-grow spinner-grow-sm"
          role="status"
        />
      </button>
    </div>
  </div>
  <div
    className="ml-auto navbar-nav"
    onKeyDown={[Function]}
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-outline-danger"
        disabled={false}
        id="last_updated"
        onClick={[Function]}
        type="button"
      >
        <span
          aria-hidden="true"
          className="spinner-grow spinner-grow-sm"
          role="status"
        />
      </button>
    </div>
  </div>
  <div
    className="ml-auto navbar-nav"
    onKeyDown={[Function]}
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-outline-info"
        disabled={false}
        id="dropdown-menu-align-right"
        onClick={[Function]}
        type="button"
      >
        <span
          aria-hidden="true"
          className="spinner-grow spinner-grow-sm"
          role="status"
        />
      </button>
    </div>
  </div>
</nav>
`;
